tests/test_vxlan_global.rb

```diff
@@ -20,23 +20,21 @@
 
 # TestVxlanGlobal - Minitest for VxlanGlobal node utility
 class TestVxlanGlobal < CiscoTestCase
+  @@clean = false # rubocop:disable Style/ClassVars
   def setup
     super
-    no_vxlan_global
-  end
-
-  def teardown
-    no_vxlan_global
-    super
+    no_vxlan_global unless @@clean
+    @@clean = true # rubocop:disable Style/ClassVars
   end
 
   def no_vxlan_global
-    config('no feature fabric forwarding')
-  end
-
-  def test_feature_on
-    VxlanGlobal.new
-    assert(Feature.fabric_forwarding_enabled?)
+    begin
+      return if !(nv_overlay_evpn_enabled?) && !(fabric_forwarding_enabled?)
+      config('no feature fabric forwarding')
+      config('no nv overlay evpn')
+    end
+  rescue
+    config('no nv overlay evpn')
   end
```

Comment:
* `nv_overlay_evpn_enabled?` is not a method or variable - I don't think this actually works...?
* This is just a cleanup any, not a test, so I would just blindly disable the feature without checking the current state
* I don't *think* the `config()` method will raise an error if it fails

In other words this may be all you need:

```
def no_vxlan_global
      config('no feature fabric forwarding')
      config('no nv overlay evpn')
end
```